wiki/Twitter/pl.md

```diff
@@ -1,17 +1,24 @@
----
-outdated_translation: true
----
-
 # Twitter
 
-## My także tweetujemy!
+osu! posiada kilka kont na Twitterze; każde z nich pełni inne funkcje. Pomimo niewielkiej aktywności oferują one wystarczające informacje o tym, co dzieje się w świecie osu!.
+
+## Serwisy
+
+| Awatar | Nazwa | Opis |
+| :-: | :-: | :-- |
+| ![osu!status avatar](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Informacje dotyczące problemów ze stroną lub [Bancho](/wiki/Bancho_(server)). |
```

Comment:
Compared to the old version, all images seem to have had their description text reverted to English and I'm not sure why. For instance, this should be:

```suggestion
| ![Awatar konta osu!](img/osustatus.jpg) | [@osustatus](https://twitter.com/osustatus) | Informacje dotyczące problemów ze stroną lub [Bancho](/wiki/Bancho_(server)). |
```

This text may seem unimportant, but it will actually be shown if the image fails to load, and be used as alt text for the image, which is important for accessibility. Refer to [commonmark spec](https://spec.commonmark.org/0.30/#images) for more info.

The same obviously goes for all of the other image links in this article.

Reply:
"Awatar konta osu!status"